examples/example_notebooks/tensorforce_example.ipynb

```diff
@@ -0,0 +1,577 @@
+    "ax2.set_ylabel('episode-length', fontsize=22)\n",
+    "ax2.tick_params(axis=\"y\", labelsize=15) \n",
+    "ax2.tick_params(axis=\"x\", labelsize=15) \n",
+    "\n",
+    "ax2.grid(True)\n",
+    "\n",
+    "plt.show()\n",
+    "\n",
+    "print('number of episodes during training: ', len(rewards))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Saving the agents trained model, makes it avaible for a seperate evalutation and further usage."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# uncomment the code and execute cell to save the trained agent\n",
+    "# agent_path = 'saved_agents'\n",
+    "# agent_name = 'dqn_agent_tensorforce'\n",
+    "# runner.agent.save(directory=agent_path, filename=agent_name)\n",
+    "# print('\\n agent saved \\n')\n",
+    "# runner.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Evaluating the trained agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.1 Loading a model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If a previously saved agent is available, it can be restored by using the runner to load the model with the **.load()** function. To load the agent it is necessary to pass the directory, the filename, the environment, and the agent configuration used for the training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tensorforce import Agent\n",
+    "\n",
+    "agent_path = 'saved_agents'\n",
+    "agent_name = 'dqn_agent_tensorforce'\n",
+    "\n",
+    "dqn_agent = Agent.load(\n",
+    "    directory=agent_path,\n",
+    "    filename=agent_name,\n",
+    "    environment=tf_env,\n",
+    "    **agent_config\n",
+    ")\n",
+    "print('\\n agent loaded \\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.3 Evaluating the agent\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use the trained agent as a controller, a typical loop to interact with the environment can be used, which is displayed in the cell below. Here it is necessary to define a new test-environment with the **set_env()** function, which disables the episode-time-limit for the agent.\n",
+    "\n",
+    "Now the agent takes the observations from the environment and reacts with an action, which is used to control the environment. To get an impression of how the trained agent performs, the trajectory of the control-states can be observed. A live-plot will be displayed in a jupyter-notebook. If you are using jupyter-lab, the following cell could cause problems regarding the visualization."
```

Comment:
Can we have a workaround for jupyter lab users?

Reply:
After my research the possible solutions using:
%matplotlib widget or some other matplotlib extensions did'nt work as well. I think if the users using jupyterlab it should not be a problem using a jupyter notebook because there is a point the drop down menu where you can the notebook you work in jupyterlab directly open in a notebook.
So i would leave it with the hint, that there will be problems for a live plot